scripts/jest.sh

```diff
@@ -0,0 +1,6 @@
+set -e -x
+
+trap "exit" INT
+
+jest --maxWorkers=2 --detectOpenHandles --forceExit \
```

Comment:
+1 for `--detectOpenHandles` - I think this has caused some strangeness in the past running the test suite in Docker...

Reply:
FWIW, I didn't add this flag, it was already in the project before this change.